gtk/docs.md

```diff
@@ -1 +1,502 @@
+`ContainerExt::child_set` or `ContainerExt::child_set_valist`.
+To obtain the value of a child property, use
+`ContainerExt::child_get_property`, `ContainerExt::child_get` or
+`ContainerExt::child_get_valist`. To emit notification about child property
+changes, use `Widget::child_notify`.
+
+# `Container` as `Buildable`
+
+The `Container` implementation of the `Buildable` interface supports
+a `<packing>` element for children, which can contain multiple `<property>`
+elements that specify child properties for the child.
+
+Since 2.16, child properties can also be marked as translatable using
+the same “translatable”, “comments” and “context” attributes that are used
+for regular properties.
+
+Since 3.16, containers can have a `<focus-chain>` element containing multiple
+`<widget>` elements, one for each child that should be added to the focus
+chain. The ”name” attribute gives the id of the widget.
+
+An example of these properties in UI definitions:
+
+```text
+<object class="GtkBox">
+  <child>
+    <object class="GtkEntry" id="entry1"/>
+    <packing>
+      <property name="pack-type">start</property>
+    </packing>
+  </child>
+  <child>
+    <object class="GtkEntry" id="entry2"/>
+  </child>
+  <focus-chain>
+    <widget name="entry1"/>
+    <widget name="entry2"/>
+  </focus-chain>
+</object>
+```
+
+# Implements
+
+[`ContainerExt`](trait.ContainerExt.html), [`WidgetExt`](trait.WidgetExt.html)
+<!-- trait ContainerExt -->
+Trait containing all `Container` methods.
+
+# Implementors
+
+[`Bin`](struct.Bin.html), [`Box`](struct.Box.html), [`Container`](struct.Container.html), [`Fixed`](struct.Fixed.html), [`FlowBox`](struct.FlowBox.html), [`Grid`](struct.Grid.html), [`HeaderBar`](struct.HeaderBar.html), [`Layout`](struct.Layout.html), [`ListBox`](struct.ListBox.html), [`MenuShell`](struct.MenuShell.html), [`Notebook`](struct.Notebook.html), [`Paned`](struct.Paned.html), [`ToolItemGroup`](struct.ToolItemGroup.html), [`ToolPalette`](struct.ToolPalette.html), [`Toolbar`](struct.Toolbar.html), [`TreeView`](struct.TreeView.html)
+<!-- trait ContainerExt::fn add -->
+Adds `widget` to `self`. Typically used for simple containers
+such as `Window`, `Frame`, or `Button`; for more complicated
+layout containers such as `Box` or `Grid`, this function will
+pick default packing parameters that may not be correct. So
+consider functions such as `BoxExt::pack_start` and
+`Grid::attach` as an alternative to `ContainerExt::add` in
+those cases. A widget may be added to only one container at a time;
+you can’t place the same widget inside two different containers.
+
+Note that some containers, such as `ScrolledWindow` or `ListBox`,
+may add intermediate children between the added widget and the
+container.
+## `widget`
+a widget to be placed inside `self`
+<!-- trait ContainerExt::fn child_notify -->
+Emits a `Widget::child-notify` signal for the
+[child property](child-properties)
```

Comment:
But we might just remove this item because child property support is not there anyway.